src/styles/primeng-theme/components/_datatable.scss

```diff
@@ -1,7 +1,13 @@
-.ui-datatable {
+@import "variables";
+
+.ui-datatable,
+.ui-table {
```

Comment:
We need to break this into 2 files, is a scss file for each component, so we need one for the new table component and datatable stays until we deprecated it.